components/mapbox/actions/create-tileset/create-tileset.mjs

```diff
@@ -0,0 +1,99 @@
+import mapbox from "../../mapbox.app.mjs";
+import fs from "fs";
+import FormData from "form-data";
+
+export default {
+  key: "mapbox-create-tileset",
+  name: "Create Tileset",
+  description: "Uploads and creates a new tileset from a data source. [See the documentation](https://docs.mapbox.com/api/maps/mapbox-tiling-service/)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    mapbox,
+    username: {
+      type: "string",
+      label: "Username",
+      description: "The Mapbox username of the account for which to create a tileset source",
+    },
+    tilesetName: {
+      type: "string",
+      label: "Tileset Name",
+      description: "A unique name for the tileset source to be created. Limited to 32 characters. The only allowed special characters are - and _.",
```

Comment:
```suggestion
      description: "A unique name for the tileset source to be created. Limited to 32 characters. The only allowed special characters are `-` (hyphen) and `_` (underscore).",
```
I think this would be a slight help for less attentive users